Clean-code-cookbook-guide.html

```
@@ -0,0 +1,1110 @@
+        <p>新しい機能を実装する前には、次のような判断をするとよいでしょう。</p>
+
+        <div class="diagram-card">
+          <div class="diagram-wrap" id="diagram-4"><div class="diagram-loading">図を読み込み中...</div></div>
+          <div class="diagram-caption">図4: Tidy Firstの意思決定フロー</div>
+        </div>
+
+        <h4>コード例（マジックナンバーの整頓）</h4>
+        <div class="code-block">
+          <div class="code-label"><i class="ti ti-brand-python"></i>Python</div>
+          <pre><code><span class="cmt"># Before</span>
+if user.age > 17:
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)